empty-plugin/src/main/resources/archetype-resources/pom.xml

```diff
@@ -4,16 +4,16 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.4</version>
+        <version>3.43</version>
```

Comment:
Seems that the tests fail with 3.33 and later. Still not quite sure why.

Reply:
Because of https://github.com/jenkinsci/plugin-pom/pull/149: the tests pass with Surefire 2.x and fail with 3.x. But only under certain conditions inside the Docker container.

Reply:
Confirmed at least locally that if you merge in #44 (throwing away the `Jenkinsfile` modification here) the failure is resolved, meaning it was just an artifact of the Docker container being set up in a weird way and this somehow confusing the Surefire booter (or something).